portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,42 @@
+body {
+    background-image: url("/images/background.jpg");
+    background-repeat: repeat-x;
+}
+#profile {
+    /* padding:0px; */
```

Comment:
It is usually better to delete commented code when committing it.

Reply:
Thanks for letting me know! I removed those comments in the most recent commit, I'll keep this in mind in the future